codex/stores/repostore.nim

```diff
@@ -64,6 +64,12 @@ iterator items*(q: BlockExpirationIter): Future[?BlockExpiration] =
 func totalUsed*(self: RepoStore): uint =
   (self.quotaUsedBytes + self.quotaReservedBytes)
 
+func available*(self: RepoStore): uint =
+  return self.quotaMaxBytes - self.totalUsed
+
+func available*(self: RepoStore, bytes: uint): bool =
```

Comment:
I would probably call this `hasAvailable()`, to differentiate from the `available` property above.